cypress/integration/inventory/export.spec.js

```diff
@@ -40,12 +36,31 @@ describe('inventory: exports', () => {
   });
 
   it('C196755 verifies search result counts and selected counts', () => {
-    cy.do([
-      InventorySearch.searchByEffectiveLocation(),
-      InventorySearch.getSearchResult(0).find(Checkbox()).click(),
-      InventorySearch.getSearchResult(2).find(Checkbox()).click(),
-    ]);
+    const checkedResult = 2;
 
-    InventorySearch.verifySelectedRecords(2);
+    InventorySearch.byEffectiveLocation();
+    InventorySearch.selectResultCheckboxes(checkedResult);
+    InventorySearch.verifySelectedRecords(checkedResult);
+  });
+
+  it('C9287 verifies export CQL instances', () => {
+    InventorySearch.byLanguage();
+    InventorySearch.byKeywords();
+    InventorySearch.byEffectiveLocation();
+    InventorySearch.saveCQLQuery();
+
+    // Need time for download file TODO: think about how it can be fixed
+    cy.wait(Cypress.env('downloadTimeout'));
+
+    FileManager.findDownloadedFilesByMask('SearchInstanceCQLQuery*')
```

Comment:
you have 2 places with 90% same code. Try to move it to one method with differences in parameters(you can use functions as parameter too)

Reply:
I think tests will be more dificult to understanding if we will have only one function with a lot of params.

I am not sure about this

`FileManager.findDownloadedFilesByMask('SearchInstanceCQLQuery*')
      .then((downloadedFilenames) => {
        const lastDownloadedFilename = downloadedFilenames.sort()[downloadedFilenames.length - 1];
        InventoryActions.verifySaveCQLQueryFileName(lastDownloadedFilename);

        FileManager.readFile(lastDownloadedFilename)
          .then((actualCQLQuery) => {
            InventoryActions.verifySaveCQLQuery(actualCQLQuery, '*', 'eng');
          });
      });
  });`

are better than this

`func(maks, funcForVerifyName, funForVerifyContent, paramsForFirstFunc, paramsForTwoFunc)`

Reply:
I see you check file name and content. Object with 2 params can be used. verifier of name has function and required parameters, verifier of content has function and required parameters too. Wrapper fileVerifier will have just {nameVerifier, contentVerifier}. 
and finally your test or this fileVerifier will have method verify with 
`  // Need time for download file TODO: think about how it can be fixed
  cy.wait(Cypress.env('downloadTimeout'));

  FileManager.findDownloadedFilesByMask(this.nameVerifier.fileNameMask)
    .then((downloadedFilenames) => {
      const lastDownloadedFilename = downloadedFilenames.sort()[downloadedFilenames.length - 1];
      this.nameVerifier.verify(lastDownloadedFilename);

      FileManager.readFile(lastDownloadedFilename)
        .then((actualContent) => {
          this.contentVerifier.verify(actualContent, this.contentVerifier.restParams);
        });
    });`